apps/test-mae3/src/main.cpp

```diff
@@ -52,7 +52,7 @@ int main() {
       MBED_ASSERT_WARN(encoder.getAngleDeg(angle));
       MBED_ASSERT_WARN(encoder.getAngularVelocityDegPerSec(speed));
 
-      Utility::Logger::printf("Angle: %f, Angular Velocity :%f\r\n", angle, speed);
+      printf("Angle: %f, Angular Velocity :%f\r\n", angle, speed);
```

Comment:
does `std::cout` work?

Reply:
yep

Reply:
Might as well use that then. What about std::endl?

Reply:
yea that works too